tests/vrf/test_vrf.py

```diff
@@ -532,10 +532,13 @@ def test_show_bgp_summary(self, duthosts, rand_one_dut_hostname, cfg_facts):
             bgp_summary_string = duthost.shell("vtysh -c 'show bgp vrf {} summary json'".format(vrf))['stdout']
             bgp_summary = json.loads(bgp_summary_string)
 
-            for info in bgp_summary.itervalues():
-                for peer, attr in info['peers'].iteritems():
+            for info in bgp_summary:
+                for peer, attr in bgp_summary[info]['peers'].iteritems():
                     prefix_count = attr['prefixReceivedCount']
-                    assert int(prefix_count) == route_count, "%s should received %s route prefixs!" % (peer, route_count)
+                    if info == "ipv4Unicast" and attr['idType'] == 'ipv6':
```

Comment:
Can you add comments to this statement. Because the if condition looks contradicting and confusing without the PR comments.

Reply:
Done